package.json

```diff
@@ -21,23 +21,22 @@
     "url": "https://github.com/AssemblyScript/assemblyscript/issues"
   },
   "dependencies": {
-    "binaryen": "101.0.0-nightly.20210904",
+    "binaryen": "102.0.0-nightly.20211027",
     "long": "^4.0.0",
-    "source-map-support": "^0.5.19",
+    "source-map-support": "^0.5.20",
     "ts-node": "^6.2.0"
   },
   "devDependencies": {
-    "@types/node": "^15.6.1",
-    "@typescript-eslint/eslint-plugin": "^4.9.1",
-    "@typescript-eslint/parser": "^4.9.1",
+    "@types/node": "^16.11.4",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "^4.33.0",
     "diff": "^5.0.0",
-    "eslint": "^7.15.0",
-    "glob": "^7.1.6",
+    "eslint": "^7.32.0",
+    "glob": "^7.2.0",
     "physical-cpu-count": "^2.0.0",
-    "source-map-support": "^0.5.19",
-    "ts-loader": "^9.2.2",
+    "ts-loader": "^9.2.6",
     "ts-node": "^6.2.0",
-    "typescript": "~4.2.4",
+    "typescript": "^4.4.4",
```

Comment:
This changed from `~` to `^`, now updating to any `4.x.x` instead of sticking to `4.4.x`. Iirc this was done because TS tends to break stuff in minor versions.

Reply:
got it